src/style.css

```
@@ -0,0 +1,157 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  margin: 0;
+  background-color: #ddd;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+/* #region Card */
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 404px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.product {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 100%;
+  margin-bottom: 17px;
+}
+
+.product__image {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+}
+
+.product__description {
+  margin: 0;
+}
+
+.product__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
```

Если поменять на h2 то надо сбросить остальные марджины
```suggestion
.product__name {
  font-weight: 500;
  font-size: 12px;
  line-height: 18px;
  color: #060b35;
  margin: 0 0 4px;
```